scripts/build.d/scotch

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -e
+
+pkgname=scotch
+pkgbranch=${VERSION:-master}
+pkgfull=${pkgname}-${pkgbranch}
+
+# # unpack (clone / pull)
+syncgit https://gitlab.inria.fr/scotch ${pkgname} ${pkgbranch}
```

Comment:
For SCOTCH, I prefer to use released tarball, which is easier to upgrade.

Reply:
If tarball is also considered, `git submodule` may be a solution for fetching a batch of source by default.

For more details let me write the idea down with this discussion thread https://github.com/solvcon/devenv/discussions/30

Reply:
Tarballs and submodules solve different things.  Tarballs may be used for offline building, and supports separate download and build.

I am not sure how well submodules play with explicit `git clone` and tarballs.  I'd guess it's either or?  If that's true, I would like to have tarballs and sacrifice submodules.

Reply:
In my opinion, if we want to use tarball, devenv need another function to deal with it.
Maybe I can file an issue to record the requirement?

Reply:
I see the created issue #32.  It looks good.